tools/csv-merger/csv-merger.go

```diff
@@ -835,6 +855,52 @@ func copyManifests() {
 	}
 }
 
+func getMetricsExporterDeployment() appsv1.DeploymentSpec {
```

Comment:
Since we're already creating the Service and such in the StorageCluster reconciliation, why are we doing this here instead of the operator code? As you say, it's not an operator, it doesn't make sense to treat it both as a separate component and a part of reconciliation.

Reply:
It's just like you install Rook via CSV and CephCluster(and it's services) via reconcile.

I kept deployment via CSV as it'll now be centrally managed along with other components (no need to separately reconcile deployment) via OLM. 

Also, it doesn't really fit into OCS reconcile because it's not a part of StorageCluster Initialization or Creation etc.
However, Service and Service Monitor fit into reconcile as they are needed wherever StorageCluster is created. Makes sense?

P.S. It's always treated as separate component. If you closely look at reconcile, Service/Service Monitor state doesn't affect OCS Operator Readiness unlike other components (which marks OCS as Not Ready during reconcile).

Reply:
Gotcha, thanks! :+1: